napari/layers/points/points.py

```diff
@@ -420,6 +428,19 @@ def __init__(
             properties=color_properties,
         )
 
+        if n_dimensional is not None:
+            warnings.warn(
+                trans._(
+                    "'n_dimensional' is deprecated and will be removed in napari v0.5.0, "
+                    "use 'out_of_slice_display' instead."
+                ),
+                DeprecationWarning,
+                stacklevel=2,
+            )
+            self._n_dimensional = n_dimensional
```

Comment:
```suggestion
            self._out_of_slice_display = n_dimensional
```

As `n_dimensional` and `out_of_slice_display` are equivalent, shouldn't we always only store `self._out_of_slice_display`?

Reply:
Yup, nice catch!